.github/workflows/ci.yml

```diff
@@ -6,37 +6,37 @@ on:
   workflow_dispatch:
 
 jobs:
-#  test_powershell:
-#    name: WindowsPowerShell
-#    runs-on: windows-latest
-#    steps:
-#      - name: Checkout Bucket
-#        uses: actions/checkout@v3
-#        with:
-#          fetch-depth: 2
-#          path: 'my_bucket'
-#      - name: Checkout Scoop
-#        uses: actions/checkout@v3
-#        with:
-#          repository: ScoopInstaller/Scoop
-#          path: 'scoop_core'
-#      - name: Init and Test
-#        shell: powershell
-#        run: |
-#          $env:SCOOP_HOME="$(Resolve-Path '.\scoop_core')"
-#          .\scoop_core\test\bin\init.ps1
-#          .\my_bucket\bin\test.ps1
+  test_powershell:
+    name: WindowsPowerShell
```

Comment:
hmmm, I don't use nor recommend PowerShell. PWSH rules ;)

Reply:
It's good to have it for backwards compatibility I guess, doesn't really matter to me, I can remove it, again mirrored ci template from master